src/main/java/poolsheen/Storage.java

```diff
@@ -0,0 +1,127 @@
+package poolsheen;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.function.Predicate;
+import java.util.Scanner;
+
+import poolsheen.task.Task;
+import poolsheen.task.Event;
+import poolsheen.task.Deadline;
+import poolsheen.task.ToDo;
```

Comment:
Would this be better if ordered alphebetically?

Reply:
Thank you for pointing this out to me!